tests/unit/test_utils.py

```diff
@@ -234,6 +235,17 @@ def test_parse_invalid_timestamp(self):
             parse_timestamp('invalid date')
 
 
+class TestDatetime2Timestamp(unittest.TestCase):
+    def test_datetime2timestamp_naive(self):
+        self.assertEqual(
+            datetime2timestamp(datetime.datetime(1970, 1, 2)), 86400)
+
+    def test_datetime2timestamp_aware(self):
+        self.assertEqual(
+            datetime2timestamp(datetime.datetime(1970, 1, 2, tzinfo=tzutc())),
```

Comment:
Let's not use ``tzutc()`` because that is default. Cannot really check that the timezone aware's timezone is respected.